dev-python/h5py/h5py-2.10.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1 flag-o-matic
+
+DESCRIPTION="Simple Python interface to HDF5 files"
+HOMEPAGE="https://github.com/h5py/h5py https://www.h5py.org"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples mpi"
+
+RDEPEND="sci-libs/hdf5:=[mpi=,hl(+)]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]"
+
+BDEPEND="dev-python/pkgconfig
+	mpi? ( virtual/mpi )"
+
+DEPEND="dev-python/cython[${PYTHON_USEDEP}]
+	mpi? ( dev-python/mpi4py[${PYTHON_USEDEP}] )
+	doc? ( dev-python/alabaster[${PYTHON_USEDEP}] )
+	test? ( dev-python/QtPy[testlib,${PYTHON_USEDEP}] )"
+
+distutils_enable_tests pytest
+distutils_enable_sphinx docs --no-autodoc
+
+pkg_setup() {
+	use mpi && export CC=mpicc
+}
+
+python_prepare_all() {
+	append-cflags -fno-strict-aliasing
+	distutils-r1_python_prepare_all
+}
+
+python_configure() {
+	esetup.py configure $(usex mpi --mpi '')
+}
+
+python_install_all() {
+	local DOCS=( README.rst AUTHORS ANN.rst )
+	use examples && DOCS+=( examples )
```

Comment:
I think it'd be better to just put DOCS as global where it belongs (before *pkg_setup*) and just call `use examples && dodoc -r examples/` here, after `distutils-r1_python_install_all`.